lib/rules/color-hex-alpha/README.md

```diff
@@ -0,0 +1,66 @@
+# color-hex-alpha
+
+Require or disallow alpha channel for hex colors.
+
+<!-- prettier-ignore -->
+```css
+a { color: #ffff }
```

Comment:
[suggestion] How about changing the alpha channel character for clarity and consistency with the following code examples? ⬇️ 

```suggestion
a { color: #fffa }
```